dashboard/config/scripts_json/allthethings.script_json

```diff
@@ -3969,6 +3969,9 @@
       "activity_section_position": 4,
       "assessment": false,
       "properties": {
+        "level_keys": [
+          "gamelab free play"
+        ]
```

Comment:
(also for my understanding) are these changes auto-generated, or something we need?